scripts/scripts/japanese_to_romaji.py

```
@@ -38,17 +41,33 @@ class RomajiResponse(BaseModel):
     Returns:
         str: The converted Romaji text
     """
     client = get_global_gemini_client()
-    prompt = SYSTEM_PROMPT
+
+    with accumulated_words_lock:
+        if accumulated_words:
+            prompt = SYSTEM_PROMPT.format(glossary_section=(
+                "See the glossary for words that have already been converted.\n"
+                "Follow the glossary to avoid converting the same word differently.\n"
+                f"Glossary: {", ".join(accumulated_words)}"
```

Pylint (3.3.7)

[error] Parsing failed: 'f-string: expecting '}' (scripts.japanese_to_romaji, line 66)'

(E0001)